src/Constants.ts

```diff
@@ -1 +1,3 @@
 export const APP_NAME = 'lfp-appeals-frontend';
+
+export const GRANT_TYPE = 'refresh_token';
```

Comment:
The name of the variable seems rather vague, especially without the context of this PR